src/scripts/home.js

```diff
@@ -226,9 +226,11 @@ function delete_profile(profile) {
   req_card.remove();
 
   // Remove profile from list and save list
-  PROFILE_LIST = PROFILE_LIST.filter(
-    (curr_prof) => curr_prof.id !== profile.id
-  );
+  const rmv_idx = PROFILE_LIST.indexOf(profile);
+  console.log("rmv idx: " + rmv_idx);
```

Comment:
Can't this be deleted

Reply:
i removed it while resolving merge conflicts, it hasnt been pushed yet